src/plugins/hidsim/readme.org

```diff
@@ -0,0 +1,113 @@
+* hidsim
+This plugin provides functionality to inject events of human-interface devices (HID) into the Xen guest under investigation by utilizing the [[https://wiki.qemu.org/Documentation/QMP][QEMU machine protocol (QMP)]].
+It can either replay pre-recorded and normalized HID-events -- evdev-events to be more specifically --, which were gathered by the tool [[file:/usr/local/src/drakvuf/src/helpers/hiddump/][./src/helpers/hiddump]] or simulate random mouse movements.
```

Comment:
`file:/usr/local/src/drakvuf/src/helpers/hiddump/` is broken on GitHub

Reply:
Corrected!